tdvt/tdvt/exprtests/standard/expected.setup.cast.str.14.txt

```diff
@@ -0,0 +1,163 @@
+<results>
+  <test name='STR([int1])'>
+    <sql>
+      SELECT CAST(`Calcs`.`int1` AS STRING) AS `TEMP_Test__2617331766__0_`&#10;FROM `tactile-pulsar-824`.`TestV1`.`Calcs` `Calcs`&#10;GROUP BY 1&#10;    </sql>
+    <query-time>
+      527.000&#10;    </query-time>
```

Comment:
Remove the all the `<sql>` and `<query-time>` elements from the expected file to match the others.

Reply:
Removed